src/sass/public/UI/header/_header.scss

```
@@ -0,0 +1,176 @@
+@use 'base/variables' as v;
+@use 'base/mixins' as m;
+
+.header {
+    box-shadow: 0px 10px 15px 0px v.$gray;   
```

```suggestion
    box-shadow: 0px 6px 15px 0px v.$gray-500;   
```
reduce the box shadow